bluebutton-data-model-codegen/src/main/java/gov/hhs/cms/bluebutton/data/model/codegen/MappingSpec.java

```diff
@@ -43,6 +43,12 @@
 	private String lineTable;
 	private List<String> headerEntityTransientFields;
 	private List<RifField> headerEntityAdditionalDatabaseFields;
+	private boolean hasInnerJoinRelationship = false;
+	private List<List<String>> innerJoinRelationship;
+	private boolean hasParentRelationship = false;
```

Comment:
I recommend handling all field initialization in one place, and -- generally -- the constructor is the best choice for that. Can you adjust the init of all these new variables?